app/app.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+const {getTopics} = require('../controllers/controllers');
+
+app.use(express.json());
```

Comment:
We do not need app.use(express.json()); yet, and thus it should not be in here at this point!